src/test/resources/features/release_version_default.feature

```diff
@@ -0,0 +1,37 @@
+#
+#  Copyright 2020 Marco Vermeulen
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+Feature: Release Version Validation
+
+  Background:
+    Given the Consumer for candidate java is making a request
+    And the Consumer has a valid Auth Token
+
+  Scenario: The Version is marked as a default
```

Comment:
It looks like this test is setting default as `false` but the scenario headline says something else. Could we have a scenario for `true` and `false`?

Perhaps we could have the scenario headline as something like `A Version is marked as default explicitly`, then another one where `A Version is not marked as default explicitly`.

Then a third case would be `A Version is not marked as default implicitly` where we test the case where the explicit `default` is omitted in the request.